modules/preview-react/status-indicator/README.md

```diff
@@ -0,0 +1,13 @@
+# Canvas Kit Status Indicator
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/preview-react/README.md">
+  <img src="https://img.shields.io/badge/PREVIEW-beta-blueviolet" alt="PREVIEW: Beta" />
+</a>  This component is work in progress and currently in preview.
+
+Status Indicators help the user quickly identify the status of a task, action, or page element.
+
+View the
+[documentation for Status Indicator](https://workday.github.io/canvas-kit/?path=/docs/preview-status-indicator-react--basic)
+on Storybook.
+
+[> Workday Design Reference: Status Indicator](https://design.workday.com/components/)
```

Comment:
@jamesfan is this still valid?

Reply:
I would actually remove that `> Workday Design Reference: Status Indicator` link entirely given that we're only documenting the Status Indicator in the Main package on the site (for now).

```suggestion
```